docs/docs/core/permissions/how-they-work.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: How they work
+tags:
+  - permissions
+  - RBAC
+---
+
+Talk about this stuff and what it means (probably needs Pierre to give me a 10min recap)
+
+```ts
+const permissions = {
+  create: [{ action: 'plugin::content-manager.explorer.create', subject: null }],
```

Comment:
@petersg83

Would you be able to give a small paragraph how the `string` works? and what a `subject` is please? From my understanding so far, `subject` only applies to `content-types` but that's probably wrong?

Reply:
on it

Reply:
Done, what do you think?

Reply:
They are really good, thank you and well done 🙇🏼‍♀️